### PR TITLE
Add some information on creating shaders

### PR DESCRIPTION
**Pull request by stirante**
It's still not complete, but at least it's the beginning of a page.

You might want to host the debug shader on your server, but I don't mind having it on my own.

### CODE REVIEW

File: knowledge/shaders.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+For more advanced debugging, you can use the debug shader I wrote [based on this shader](http://mew.cx/drawtext/drawtext.html).
+Right now this shader will display values of the color passed to the shader. To display another value, change line 70 in hlsl shader to
+```
+int ascii = getFloatCharacter( cellIndex, <float4 vector here> );
+```
+GLSL version of debug shader may crash Minecraft, use only for debugging.
+
+[Download debug shader](http://files.stirante.com/debugShader.zip)
+
+![](/assets/img/debugShader.gif)
```
Comment (SirLich) [inline knowledge/shaders.md]: Lets index this better. Can we do /assets/images/knowledge/shaders/debugShader.gif?

Just so as we add more media it doesn't get overwhelming

File: knowledge/shaders.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: Shaders
+parent: Knowledge
+---
+
+# Shaders
+
```
Comment (SirLich) [inline knowledge/shaders.md:8]: Maybe include a section about how shaders can only be used with the RTX Beta? Just so people don't get confused?

Comment (stirante) [reply knowledge/shaders.md:8]: But you can use shaders in any version of Minecraft. I also verified, that shaders work on android version of Minecraft.

File: knowledge/shaders.md (hunk 1 above)
Comment (stirante) [reply knowledge/shaders.md]: Done

**Pull request merged**